### ISSUE TITLE
Support for dates

### ISSUE THREAD

**Issue by sbarzowski** [getguesstimate/guesstimate-app#997]
The tool is really useful for estimating how long things will take considering multiple factors. It would be really nice if such things could be presented as dates (rather than just the numbers of days or similar).  I imagine it being used mostly as the last step. 

I imagine it could be implemented through:
a) a presentation option of a cell to treat the values as the number of days since some arbitrarily defined epoch (e.g. 01-01-1970) and format them as dates
b) a builtin function which converts a date in human format to days-since-epoch representation

**Comment by OAGr** [comment]
I've been thinking about this for a related application. I'm not sure what the ideal format would be. One issue with ISO / time since 1970 is that, of course, it's only from 1970, so if people wanted to estimate things in history there would be issues. Do you have ideas about what format would be ideal?

That said, I think this would be a decent amount of work, so I think it's unlikely I'll do it personally.

**Comment by sbarzowski** [comment]
> One issue with ISO / time since 1970 is that, of course, it's only from 1970, so if people wanted to estimate things in history there would be issues.

Not necessarily. You could have a negative number of "days since epoch" to represent the dates before that. The 0-point doesn't really matter it just needs to be set to something.

**Comment by OAGr** [comment]
That's a good point. I've been thinking about this more, may do it eventually.

**Comment by fire** [comment]
Did you think about this more?

**Comment by OAGr** [comment]
No real update yet.

I've been playing with this with a new experimental application, Estiband.  I think doing it right is a bit tricky, it depends on how generic we want to be. You can go [here](https://hedoniccalculator.com/dist-builder) and change "Unit type" to change the unit to one of date.

This work definitely seems doable, but tricky. Dates would involve changing several parts, input, visualization, calculation.

I'm working to set up a nonprofit and move Guesstimate into it, then continue development with Guesstimate a bit. The other nonprofit work would be pretty integrated, though somewhat separate.
